<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeywordsDialog</class>
 <widget class="QDialog" name="KeywordsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Metadata Keywords</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="subjectLayout">
     <item>
      <layout class="QVBoxLayout" name="_2">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Keywords</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="_3">
         <item>
          <widget class="SpotlightWidget" name="keyword"/>
         </item>
         <item>
          <widget class="QToolButton" name="addButton">
           <property name="styleSheet">
            <string notr="true">QToolButton {border: none;}</string>
           </property>
           <property name="text">
            <string>+</string>
           </property>
           <property name="icon">
            <iconset resource="../../FileManager/FileManager.qrc">
             <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="removeButton">
           <property name="styleSheet">
            <string notr="true">QToolButton {border: none;}</string>
           </property>
           <property name="text">
            <string>-</string>
           </property>
           <property name="icon">
            <iconset resource="../QExiv2.qrc">
             <normaloff>:/images/delete.png</normaloff>:/images/delete.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListView" name="keywordListView"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SpotlightWidget</class>
   <extends>QLineEdit</extends>
   <header>SpotlightWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../FileManager/FileManager.qrc"/>
  <include location="../QExiv2.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>KeywordsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>KeywordsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
